staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -1125,6 +1125,10 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 				klog.V(10).Infof("reconcileLoadBalancer for service (%s)(%t): lb backendpool - found wanted backendpool. not adding anything", serviceName, wantLb)
 				foundBackendPool = true
 
+				if !az.useStandardLoadBalancer() || !az.EnableMultipleStandardLoadBalancers {
```

Comment:
This says 'not useStandardLoadBalancer' is this intended?  What kind of load balancer are we using?  Do we use some that are standard and some that are not standard?

Reply:
We use a standard loadbalancer.